extensions/rocksdb-repos/RocksDbRepository.cpp

```diff
@@ -20,22 +20,22 @@ using namespace std::literals::chrono_literals;
 
 namespace org::apache::nifi::minifi::core::repository {
 
-void RocksDbRepository::printStats() {
+std::optional<RepositoryMetricsSource::RocksDbStats> RocksDbRepository::getRocksDbStats() const {
+  RocksDbStats stats;
   auto opendb = db_->open();
   if (!opendb) {
-    return;
+    return stats;
   }
-  std::string key_count;
-  opendb->GetProperty("rocksdb.estimate-num-keys", &key_count);
 
   std::string table_readers;
   opendb->GetProperty("rocksdb.estimate-table-readers-mem", &table_readers);
+  stats.table_readers_size = std::stoull(table_readers);
```

Comment:
Are the possible exceptions thrown by `std::stoull` properly handled?

Reply:
Good point, updated in 453104bb9514c478ef02f182ae1f4ccbd7572aaf